src/MOD_DYN.F90

```diff
@@ -18,13 +18,13 @@ MODULE MOD_DYN
     integer       :: lutype  = 2
     real(kind=WP) :: droptol = 1.e-8
 !!! PARMS Solver
-    real(kind=WP) :: soltol  = 1e-10  ! default for PARMS
-    logical       :: use_parms = .TRUE.
-!!!
-!!! Sergey's Solver
-!   real(kind=WP)  :: soltol  = 1e-5  ! default for PARMS
-!   logical        :: use_parms = .FALSE.
+!    real(kind=WP) :: soltol  = 1e-10  ! default for PARMS
```

Comment:
This can be changed, PARMS is gone.

Reply:
The rest looks good, mostly multio and some OMP changes.

Reply:
yes, I will try to remove the unnecessary changes (also some commented out stuff from Suvi), but not exactly sure how to do it without breaking git ;) 

The thing I was wondering about is the additional MPI barriers, I am not sure if they are needed, if they harm performance etc.